charts/hono/templates/hono-service-command-router/hono-service-command-router-secret.yaml

```diff
@@ -60,16 +60,16 @@ stringData:
             socketTimeout: 5000
             connectTimeout: 5000
           {{- end }}
-      registration:
       {{- if .Values.deviceRegistryExample.enabled }}
-        name: {{ printf "Hono %s" $args.component | quote }}
-        host: {{ .Release.Name }}-service-device-registry
-        port: 5671
-        credentialsPath: /etc/hono/adapter.credentials
-        trustStorePath: /etc/hono/trusted-certs.pem
-        hostnameVerificationRequired: false
+      registration:
+        {{- include "hono.deviceRegistryExampleClientConfig" $args | nindent 8 }}
+      tenant:
+        {{- include "hono.deviceRegistryExampleClientConfig" $args | nindent 8 }}
       {{- else }}
+      registration:
         {{- required ".Values.adapters.deviceRegistrationSpec MUST be set if example Device Registry is disabled" .Values.adapters.deviceRegistrationSpec | toYaml | nindent 8 }}
+      tenant:
+        {{- required ".Values.adapters.tenantSpec MUST be set if example Device Registry is disabled" .Values.adapters.tenantSpec | toYaml | nindent 8 }}
```

Comment:
can we also please add configuration for Quarkus logging as we do for e.g. the AMQP adapter?